backend/game/cards/keep_cards/health_manipulation_cards/it_has_a_child.py

```diff
@@ -0,0 +1,18 @@
+from game.cards.card import Card
+from game.cards.keep_card import KeepCard
+from game.values import constants
+
+
+class ItHasAChild(KeepCard):
+    def __init__(self):
+        super(KeepCard, self).__init__("It Has a Child", 7,
+                      "If you reach 0[health] discard all your cards and lose all you [star]. Gain 10[health] and continue playing outside Tokyo",
+                      "")
+
+    def special_effect(self, player_that_bought_the_card, other_players):
+        if player_that_bought_the_card.energy == constants.ZERO_ENERGY:
```

Comment:
I think this should be `player_that_bought_the_card.current_health` not `player_that_bought_the_card.energy`

Reply:
@cvennel is right. Based on the description, this should take affect when they player's health reaches zero.

Reply:
moved "it has a child" to its own PR